apps/storybook/src/overlays/AlertDialog.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { AlertDialog, Button, Flex } from "@optiaxiom/react";
+import { expect, screen, userEvent, within } from "@storybook/test";
+import { useState } from "react";
+
+const meta: Meta<typeof AlertDialog> = {
+  argTypes: {
+    action: { control: "text" },
+    cancel: { control: "text" },
```

Comment:
don't need to specify these manually - storybook will pick them up automagically